admin.php

```diff
@@ -1,5 +1,201 @@
 <?php
 
+
+/**this function is to get subtitle data for the about_me section from the database
+ *
+ * @param $db array of subtitle field from about me table from db
+ *
+ * @return array of subtitle field
+ */
+function getAboutMeSubtitleFromDatabase($db) {
+    $query = $db->prepare("SELECT `id`,`subtitle` FROM `about_me` WHERE `deleted` = 0;");
+    $query->execute();
+    return $query->fetchAll();
+}
+
+
+/**this function is to return the names of each subtitle in about_me section into a drop-down list
+ *
+ * @param $subtitleArray array of values in subtitle field from about me table
+ *
+ * @return string the name of each subtitle
+ */
+function listingSubtitles($subtitleArray) {
+    $results = " ";
+    foreach ($subtitleArray as $value) {
+        $results .=  "<option value='" . $value['id'] . "'>" . $value['subtitle'] . "</option>";
+    }
+    return $results;
+}
```

Comment:
I would insert `$results` into a string of your form and then return the entire string.
So the function generates a form with options rather than just options...

Reply:
thank you, Nic. I'm going to leave this as it is for now because I've kept it consistent throughout - as you've noted below ! I realise it's more efficient to use the variable in a string like you've suggested.